### PR TITLE
Drop turbolinks dependency

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Drops dependency on turbolinks, Obsoletes it in the `scl` but leaves it out in the repo.

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
> This is missing the addition of coffee-rails that's in assets.

Like this in the `assets` should be enough, right?

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -98,8 +98,6 @@ Requires: %{?scl_prefix}rubygem(rails-i18n) >= 5.0
 Requires: %{?scl_prefix}rubygem(rails-i18n) < 6.0
 Requires: %{?scl_prefix_ror}rubygem(i18n) >= 1.1
 Requires: %{?scl_prefix_ror}rubygem(i18n) < 2.0
-Requires: %{?scl_prefix_ror}rubygem(turbolinks) >= 2.5.4
```
Comment (tbrisker) [inline packages/foreman/foreman/foreman.spec]: should we also obsolete this?

File: packages/foreman/tfm/tfm.spec [hunk 2]
```diff
@@ -61,6 +61,7 @@ Obsoletes: ruby193-ruby-wrapper
 
 # Obsolete packages no longer carried within this SCL to both clean up
 # and to ensure smooth upgrades when old packages deps aren't satisified
+Obsoletes: %{scl_prefix_ror}rubygem-jquery-turbolinks < 3.0
```
Comment (ekohl) [inline packages/foreman/tfm/tfm.spec]: IMHO it's not very clean to obsolete something from another collection here. However, this particular example is also from the regular scl. It's also custom to pin on `Version-(Release+1)`. See https://github.com/theforeman/foreman-packaging/tree/rpm/develop#howto-removing-a-package
```suggestion
Obsoletes: %{scl_prefix}rubygem-jquery-turbolinks < 2.1.0-9
```

File: package_manifest.yaml [hunk 3]
```diff
@@ -126,7 +126,6 @@ foreman_scl_packages:
     rubygem-httpclient: {}
     rubygem-http-cookie: {}
     rubygem-ipaddress: {}
-    rubygem-jquery-turbolinks: {}
```
Comment (ekohl) [inline package_manifest.yaml]: Please also remove the sources in `packages/foreman/rubygem-jquery-turbolinks`

File: packages/foreman/foreman/foreman.spec (hunk 1 above)
Comment (ekohl) [reply packages/foreman/foreman/foreman.spec]: No, obsoleting something from a different collection isn't correct IMHO.

File: packages/foreman/foreman/foreman.spec [hunk 4]
```diff
@@ -310,8 +306,7 @@ BuildRequires: %{?scl_prefix}npm(react-intl) < 3.0.0
 # end package.json dependencies BuildRequires
 
 # start specfile assets BuildRequires
-BuildRequires: %{?scl_prefix_ror}rubygem(jquery-turbolinks) >= 2.1
-BuildRequires: %{?scl_prefix_ror}rubygem(jquery-turbolinks) < 3.0
+BuildRequires: %{?scl_prefix_ror}rubygem(coffee-rails) >= 4.2.2
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec]: You're missing the upper bounds. https://github.com/ekohl/theforeman-plumbing/blob/master/fm-packaging-update-deps would have done all of this for you and made sure that in the future it'd also remain consistently sorted.

**Comment by ezr-ondrej** [comment]
Any idea why it is looking for `tfm-rubygem-jquery-turbolinks = 0:2.1.0-8.el7`?

**Comment by ezr-ondrej** [comment]
Ok, it passes if I obsolated the `-doc` packages. So what to do about it? Is it acceptable to obsolete both? :thinking:

**Comment by zjhuntin** [comment]
I think it's acceptable to obsolete both, may seem stringent, but it covers all the bases.

**Comment by ekohl** [comment]
Anyone from delivery minds merging and doing the cleanup?

**Comment by tbrisker** [comment]
Looks like this needs a rebase please

**Comment by ezr-ondrej** [comment]
@tbrisker done :)

**Pull request merged**